src/components/MapController/MapController.js

```diff
@@ -19,51 +38,98 @@ var MapController = React.createClass({
   propTypes: {
     variable: React.PropTypes.string,
     meta: React.PropTypes.array,
+    comparand: React.PropTypes.string,
+    comparandMeta: React.PropTypes.array,
     onSetArea: React.PropTypes.func.isRequired,
   },
 
   mixins: [ModalMixin],
 
   /*
-   * State items also set from meta object array Includes: - dataset - wmstime -
-   * variable
+   * State items also set from meta object array 
+   * Includes:
+   *  - dataset
+   *  - wmstime
+   *  - variable
    */
   getInitialState: function () {
     return {
-      styles: 'boxfill/ferret',
-      timeidx: 0,
-      logscale: false,
+      scalarLogscale: false,
+      numberOfContours: 10,
+      contourLogscale: false,
     };
   },
-
+  
+  //this function handles user selection of all the straightforward
+  //parameters like logscale or palette.
   updateSelection: function (param, selection) {
     var update = {}; update[param] = selection;
     this.setState(update);
   },
 
+  //this function updates the timestamp displayed on the map and triggers 
+  //a rerender.
```

Comment:
Technically, this code only updates the time values in the state. The consequences of that are to update the map display and trigger a renderer. In fact, the consequences of that update could change (if other code changes), presumably, so better to state this as 'nominally ...' or 'typically ...' or omit entirely unless this consequence is critical to understanding something here.